sample_consensus/include/pcl/sample_consensus/impl/ransac.hpp

```diff
@@ -55,77 +58,184 @@ pcl::RandomSampleConsensus<PointT>::computeModel (int)
   }
 
   iterations_ = 0;
-  int n_best_inliers_count = -INT_MAX;
-  double k = 1.0;
+  unsigned int n_best_inliers_count = 0;
```

Comment:
Since this is still in "draft" phase, I'll just recommend using `std::size_t` instead of `unsigned int` for representing size. That'll make it easier to choose a better alternative for size related variables later on.

Reply:
Makes sense. Done.